src/flash/flasher.cpp

```diff
@@ -64,17 +65,28 @@ void Flasher::setVerify(bool verify)
 
 QString Flasher::stm32flashPath()
 {
-#ifdef Q_OS_OSX
-    // macdeployqt file do not put stm32flash binary in the same folder of pingviewer
-    static QString absoluteBinPath = QCoreApplication::applicationDirPath() + "/../..";
+    static const QString binaryName =
```

Comment:
I think we should check local path first because it is already there for deployments and so that deployments are using a known version of stm32flash.

Reply:
Done.